polaris-service/src/main/java/org/apache/polaris/service/admin/PolarisAdminService.java

```diff
@@ -533,6 +534,24 @@ private boolean catalogOverlapsWithExistingCatalog(CatalogEntity catalogEntity)
             });
   }
 
+  private boolean catalogLocationsContainWasb(CatalogEntity catalogEntity) {
```

Comment:
Quite a thin method and only used in one place. If we are going to make it its own method, can we at least make it more visible to test it?